pkgs/data/fonts/hubot-sans/default.nix

```diff
@@ -0,0 +1,35 @@
+{ lib, stdenv, fetchFromGitHub }:
```

Comment:
If there is no C compilation, use stdenvNoCC:

```suggestion
{ lib
, stdenvNoCC
, fetchFromGitHub 
}:
```

Reply:
Very good, I didn't know that thanks :)